.github/workflows/unit_test.yaml

```diff
@@ -8,6 +8,7 @@ on:
       - 'setup.py'
       - 'pyproject.toml'
       - '.github/workflows/**'
+  push:
 
```

Comment:
Maybe you also want to add the `paths` filters so that doc changes will not trigger the action? I remember github action has monthly runtime quota.

Reply:
Just for temporary testing, will change to `pull_request_...`.